AppConfig/Config.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AppConfig.Interfaces;
+using System.Configuration;
+
+namespace AppConfig
+{
+    public class Config : IConfig
+    {
+        public string Key => ConfigurationManager.AppSettings["APIKey"];
+
```

Comment:
Move it to an extension method or read the config in the constructor and set the properties through the indexer

Reply:
Done